scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -151,6 +154,21 @@ class FossId(
             v.takeUnless { k in secretKeys }.orEmpty()
         }
 
+    private suspend fun getProject(projectCode: String): Project? =
+        service.getProject(user, apiKey, projectCode).run {
+            when {
+                error == null && data != null -> {
+                    FossId.log.info { "Project '$projectCode' exists." }
+                    data
+                }
+                status == 0 && error == "Project does not exist" -> {
+                    FossId.log.info { "Project '$projectCode' does not exist." }
+                    null
+                }
+                else -> throw IOException("Could not get project. Additional information : $error")
```

Comment:
Just curious: Did you deliberately drop the blank lines between the `when` cases?

Reply:
I don't copy paste code :0)
Hence I typed myself your suggestion and both 'suggestions' got lost.

Fixed for both comments.

Reply:
> I don't copy paste code :0)

I think it's fine to do that, as I often spend some time on code snippets I suggest to ensure they compile; so "bigger" changes > 5 lines of code I do not type blindly.

But in any case, even taking over "test" snippets, you should review the snippet of course.